lib/src/main/java/io/ably/lib/realtime/Presence.java

```diff
@@ -511,6 +546,14 @@ public void updatePresence(PresenceMessage msg, CompletionListener listener) thr
 		}
 	}
 
+	/**
+	 * (RTP13) Presence#syncComplete returns true if the initial SYNC operation has completed for
+	 * the members present on the channel
+	 */
+	public boolean syncComplete() {
```

Comment:
In spec it is described as a method (returns). There is no problem changing it to field